plans/GOAP_STATE.md

```
@@ -203,9 +203,9 @@ Impact analysis of `d-o-hub/github-template-ai-agents` and `d-o-hub/chaotic_sema
 | Workspace version | 0.1.31 | — |
 | Latest GitHub release | v0.1.31 | verified 2026-04-30 |
 | Publishable workspace crates | 6 | all at 0.1.31 |
-| Total tests | 2,856 | — |
-| Ignored tests | 123 skipped | ceiling ≤125 |
-| `allow(dead_code)` (prod) | 0 | ≤25 | ✅ All in test/bench files (36 total) |
+| Total tests | 3,282 | — |
+| Ignored tests | 164 skipped | ceiling ≤165 |
+| `allow(dead_code)` (prod) | 38 | ≤25 | ⚠️ Over (API reserves/future features, verified 2026-05-16) |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)